src/shared/cmvision/cmvision_region.h

```diff
@@ -28,8 +28,6 @@
 #include "cmvision_threshold.h"
 #include "lut3d.h"
 
-#define CMV_DEFAULT_MAX_RUNS 100000
```

Comment:
Was this not used anywhere else? It doesnt' show up in the diff for the PR.

Reply:
no, was not used. The numbers were hard-coded before.